packages/dotcom-build-base/package.json

```diff
@@ -6,12 +6,9 @@
   "types": "src/index.ts",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "tsc": "../../node_modules/.bin/tsc --incremental",
     "clean": "npm run clean:dist && npm run clean:node_modules",
     "clean:dist": "rm -rf dist",
     "clean:node_modules": "rm -rf node_modules",
-    "build": "npm run build:node",
-    "build:node": "npm run tsc -- --module commonjs --outDir ./dist/node",
```

Comment:
**question:** you have fully removed, rather than edited, the `build:node` and `build` scripts back in here - was that intentional?

Reply:
It gets removed in this commit:  https://github.com/Financial-Times/dotcom-page-kit/pull/970/commits/61621ac88625245ee0f622d95084b1a67035f50d

Reply:
I think your response is not the answer to this question? 😊

Reply:
Oops! I will add it back in to be like the other packages.

Reply:
Great, 👍 from me once that's done!

Reply:
Done!